backend/api/views/fp.py

```diff
@@ -64,15 +64,15 @@ def new_fp():
             status=400, message="No application status provided for new FP"
         )
     sample_args = request.args
-    new_fp = FieldPartner(**data)
+    new_fp = FieldPartner(data)
```

Comment:
As a result of the new `__init()__` syntax, we can simply pass in `data` to model constructors instead of using `**data`.